tests/gocase/util/server.go

```diff
@@ -34,6 +35,10 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+var binPath = "placeholder"
+var workspace = "placeholder"
+var deleteOnExit = "false"
```

Comment:
I think we can use the flag package like https://stackoverflow.com/questions/27342973/custom-command-line-flags-in-gos-unit-tests ?

Reply:
Make sense. Let me try it out later.